src/main/java/entities/Developer.java

```diff
@@ -0,0 +1,34 @@
+package entities;
+
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
+import javax.persistence.Table;
+import java.util.List;
+import java.util.Set;
+
+@ToString
+@Entity(name = "developer")
+@Table (name = "developers")
+@Getter
+@Setter
+public class Developer extends Human {
+    @Column(name = "age")
+    private Integer age;
+    @Column(name = "is_man")
+    private Boolean isMale;
```

Comment:
Better use enum here
https://vladmihalcea.com/the-best-way-to-map-an-enum-type-with-jpa-and-hibernate/

Reply:
Saved in the table as varchar